[2025-06-20 10:33:50|16912|ERROR|__init__.py:115]  match error, info is 405 Method Not Allowed: The method is not allowed for the requested URL.
[2025-06-20 10:33:50|16912|ERROR|__init__.py:116]  Traceback (most recent call last):
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\ruanjian\Python\Python310\lib\site-packages\werkzeug\routing\map.py", line 619, in match
    raise MethodNotAllowed(valid_methods=list(e.have_match_for)) from None
werkzeug.exceptions.MethodNotAllowed: 405 Method Not Allowed: The method is not allowed for the requested URL.

[2025-06-20 10:33:59|19840|ERROR|__init__.py:115]  match error, info is 405 Method Not Allowed: The method is not allowed for the requested URL.
[2025-06-20 10:33:59|19840|ERROR|__init__.py:116]  Traceback (most recent call last):
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\ruanjian\Python\Python310\lib\site-packages\werkzeug\routing\map.py", line 619, in match
    raise MethodNotAllowed(valid_methods=list(e.have_match_for)) from None
werkzeug.exceptions.MethodNotAllowed: 405 Method Not Allowed: The method is not allowed for the requested URL.

[2025-06-20 10:35:07|24976|ERROR|__init__.py:115]  match error, info is 405 Method Not Allowed: The method is not allowed for the requested URL.
[2025-06-20 10:35:07|24976|ERROR|__init__.py:116]  Traceback (most recent call last):
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\ruanjian\Python\Python310\lib\site-packages\werkzeug\routing\map.py", line 619, in match
    raise MethodNotAllowed(valid_methods=list(e.have_match_for)) from None
werkzeug.exceptions.MethodNotAllowed: 405 Method Not Allowed: The method is not allowed for the requested URL.

[2025-06-20 10:35:51|11952|ERROR|__init__.py:115]  match error, info is 405 Method Not Allowed: The method is not allowed for the requested URL.
[2025-06-20 10:35:51|11952|ERROR|__init__.py:116]  Traceback (most recent call last):
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 844, in dispatch_request
    self.raise_routing_exception(req)
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\app.py", line 453, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
  File "D:\ruanjian\Python\Python310\lib\site-packages\flask\ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
  File "D:\ruanjian\Python\Python310\lib\site-packages\werkzeug\routing\map.py", line 619, in match
    raise MethodNotAllowed(valid_methods=list(e.have_match_for)) from None
werkzeug.exceptions.MethodNotAllowed: 405 Method Not Allowed: The method is not allowed for the requested URL.

